### DISCUSSION TITLE
Environment variable to disable Forge Version Checker?

### DISCUSSION THREAD

**Discussion by ChrisLane** [itzg/docker-minecraft-server#1984] [Q&A]
Is there an environment variable to disable the forge version checker?

**Reasons I want to disable the feature:**
1. I'm running a curseforge modpack server and so don't need any version checking since I won't update mods individually.
1. When I start my server with my VPN enabled I get `java.net.http.HttpConnectTimeoutException: HTTP connect timed out` errors in the forge version checks which brings my startup to a halt before the server is joinable.

**Comment by itzg** [comment]
Take a look at https://github.com/itzg/docker-minecraft-server#running-a-forge-server

**Reply by ChrisLane** [reply]
I don't think that answers my question. I'm running a curseforge modpack with `TYPE: AUTO_CURSEFORGE` and the forge version checking is for mod version checking, toggled by `versionCheck` in config/fml.toml.

I'm asking whether there is a setting to disable that property via my docker-compose.yml (I suspect there isn't).

**Reply by itzg** [reply]
Thanks for clarifying.

How did you determine it is the forge version check specifically that is failing?

**Reply by itzg** [reply]
...I just re-read your follow-up and noticed you mentioned the `config/fml.toml` version check. That's not directly used by the container startup software, so you'll need to overlay a modified config file using https://github.com/itzg/docker-minecraft-server#optional-plugins-mods-and-config-attach-points

**Reply by ChrisLane** [reply]
> Thanks for clarifying.
> 
> How did you determine it is the forge version check specifically that is failing?

The logs and stacktrace at the time are all prefixed with `[Forge Version Check/INFO] [ne.mi.fm.VersionChecker/]`


> ...I just re-read your follow-up and noticed you mentioned the `config/fml.toml` version check. That's not directly used by the container startup software, so you'll need to overlay a modified config file using https://github.com/itzg/docker-minecraft-server#optional-plugins-mods-and-config-attach-points

Looks like I can use this, thanks!
